examples/liquid.ini

```diff
@@ -72,6 +72,10 @@ wait_green_count = 10
 [job:zipkin]
 template = templates/zipkin.nomad
 
+# Run a custom local job, see litterbox.py
```

Comment:
We need to test this config on the CI.

We can either put it in `vagrant/provision-liquid.sh`, but that will also taint the virtualbox VMs with proverbial cats and litterboxes.

If you don't want to do that now, please open an issue for it.

Reply:
Ok, I've made an issue. https://github.com/liquidinvestigations/node/issues/118